### PR TITLE
CI build and tests added for Gaudi Notebook container

### PR DESCRIPTION
**Pull request by sharvil10**
This PR adds actions file for Gaudi notebooks container to enable CI for the container.

### CODE REVIEW

File: enterprise/redhat/openshift-ai/gaudi/docker/.actions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "RHEL_OS": ["9.2"],
+    "experimental": [true],
+    "runner_label": ["Gaudi"]
```
Comment (tylertitsworth) [inline enterprise/redhat/openshift-ai/gaudi/docker/.actions.json:4]: the label will likely be `gaudi2`

Comment (sharvil10) [reply enterprise/redhat/openshift-ai/gaudi/docker/.actions.json]: Updated

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -50,3 +50,22 @@ jobs:
         VALIDATE_PYTHON_FLAKE8: false
         VALIDATE_PYTHON_MYPY: false
         VALIDATE_PYTHON_RUFF: false
+    - name: Super-Linter
+      uses: super-linter/super-linter/slim@b4515bd4ad9d0aa4681960e053916ab991bdbe96
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # Set your fix mode variables to true
+        FIX_SHELL_SHFMT: true
```
Comment (tylertitsworth) [inline .github/workflows/lint.yaml:58]: You don't need to run super linter again

Comment (sharvil10) [reply .github/workflows/lint.yaml]: Removed the linter changes. Will add in a separate PR later.

**Review by tylertitsworth — CHANGES_REQUESTED** [review]

**Review by tylertitsworth — APPROVED** [review]
LGTM, let's wait for the CI to succeed first.

**Pull request merged**